apps/web/src/app/api/checker/regions/_checker.ts

```diff
@@ -48,6 +35,18 @@ const monitor = async (
     latency,
     url,
     region,
+    cronTimestamp,
+    // TODO: discuss how to use the metadata properly
+    // metadata: {
+    //   status: res.status,
+    //   statusText: res.statusText,
+    //   ok: res.ok,
+    //   headers: res.headers,
+    //   // body: res.body ? JSON.parse(res.body),
+    //   bodyUsed: res.bodyUsed,
+    //   redirected: res.redirected,
+    //   type: res.type,
+    // },
```

Comment:
Need to check what needs to be stored as well. The `Headers` data are always interesting. Otherwise not everything is useful.